packages/invoices/src/OutboundShipment/DetailView/reducer.test.ts

```diff
@@ -293,68 +341,73 @@ describe('DetailView reducer', () => {
       }
     });
   });
+});
 
-  it('transfers deleted lines from the draft to the deleted lines cache', () => {
+describe('DetailView reducer: deleting lines', () => {
+  it('deleted lines are tagged as such', () => {
     const lineToDelete = createLine('1');
     const state = callReducer(OutboundAction.deleteLine(lineToDelete));
 
     const line = state.draft.lines.find(({ id }) => lineToDelete.id === id);
 
-    expect(line).toBeFalsy();
-    expect(state.deletedLines[0]).toEqual(lines[0]);
-  });
-
-  it('updates an existing line when upserting', () => {
-    const lineToUpdate = createLine('1', { numberOfPacks: 999 });
-    const state = callReducer(OutboundAction.upsertLine(lineToUpdate));
-
-    // Try find the lines we just deleted
-    const line = state.draft.lines.find(({ id }) => lineToUpdate.id === id);
-    expect(line?.numberOfPacks).toBe(999);
+    expect(line).toEqual(expect.objectContaining({ isDeleted: true }));
```

Comment:
this is the case where a user has added a line -> saved -> deleted the line - so we need to tag it to send it for a delete